watertap/examples/flowsheets/full_treatment_train/flowsheet_components/chemistry/tests/test_posttreatment.py

```diff
@@ -67,28 +67,29 @@ def test_ideal_naocl_chlorination():
     ].value == pytest.approx(5.6407676871845223e-11, rel=1e-3)
 
 
+@pytest.mark.requires_idaes_solver
 @pytest.mark.component
 def test_ideal_naocl_chlorination_full_block():
     model = run_chlorination_block_example(fix_free_chlorine=True)
     assert model.fs.ideal_naocl_mixer_unit.dosing_rate.value == pytest.approx(
-        0.9504457542440085e-6, rel=1e-3
+        1.7296113311683092e-09, rel=1e-3
```

Comment:
3 order of magnitude change--not sure if this is correct due to inaccuracies in the original values used that were corrected or if this is an error itself. Without a deep dive into verifying replaced values, my guess is that this result is due to the former.

Reply:
`dh_rxn_ref` was *also* corrected by three orders of magnitude above.

There's also other changes to how the solubility equilibrium constraint is implemented in IDAES https://github.com/IDAES/idaes-pse/pull/1197

Reply:
OK, so due to the first reason I mentioned. Thanks for confirming.